pkgs/games/path-of-building/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, fetchFromGitHub, runCommand, unzip, meson, ninja, pkg-config, qtbase, qttools, wrapQtAppsHook, luajit }:
+let
+  dataVersion = "2.27.2";
+  frontendVersion = "unstable-2023-04-09";
+in
+stdenv.mkDerivation {
+  pname = "path-of-building";
+  version = "${dataVersion}-${frontendVersion}";
+
+  src = fetchFromGitHub {
+    owner = "ernstp";
+    repo = "pobfrontend";
+    rev = "9faa19aa362f975737169824c1578d5011487c18";
+    hash = "sha256-zhw2PZ6ZNMgZ2hG+a6AcYBkeg7kbBHNc2eSt4if17Wk=";
+  };
+
+  data = runCommand "path-of-building-data" {
+    src = fetchFromGitHub {
+      owner = "PathOfBuildingCommunity";
+      repo = "PathOfBuilding";
+      rev = "v${dataVersion}";
+      hash = "sha256-z5jkTa3Cfsg+A/fK8k37/bwLRDlEvWBoZhwYrOnjyjY=";
+    };
+
+    nativeBuildInputs = [ unzip ];
+  }
+  ''
+    # I have absolutely no idea how this file is generated
+    # and I don't think I want to know. The Flatpak also does this.
+    unzip -j -d $out $src/runtime-win32.zip lua/sha1.lua
+
+    # Install the actual data
+    cp -r $src/src $src/runtime/lua/*.lua $src/manifest.xml $out
+
+    # Pretend this is an official build so we don't get the ugly "dev mode" warning
+    substituteInPlace $out/manifest.xml --replace '<Version' '<Version platform="nixos"'
+    touch $out/installed.cfg
+
+    # Completely stub out the update check
+    chmod +w $out/src/UpdateCheck.lua
+    echo 'return "none"' > $out/src/UpdateCheck.lua
+  '';
+
+  nativeBuildInputs = [ meson ninja pkg-config qttools wrapQtAppsHook ];
+  buildInputs = [ qtbase luajit luajit.pkgs.lua-curl ];
+  dontWrapQtApps = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 pobfrontend $out/bin/pobfrontend
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    wrapQtApp $out/bin/pobfrontend \
+      --set LUA_PATH "$LUA_PATH" \
+      --set LUA_CPATH "$LUA_CPATH" \
+      --chdir "$data"
+  '';
+
+  meta = {
+    description = "Offline build planner for Path of Exile";
+    homepage = "https://pathofbuilding.community/";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.k900 ];
+    mainProgram = "pobfrontend";
```

Comment:
Maybe we just mark darwin as broken?

```
The Meson build system
Version: 1.0.0
Source dir: /private/tmp/nix-build-path-of-building-2.27.2-unstable-2023-04-09.drv-0/source
Build dir: /private/tmp/nix-build-path-of-building-2.27.2-unstable-2023-04-09.drv-0/source/build
Build type: native build
Project name: POB Frontend
Project version: undefined
C++ compiler for the host machine: clang++ (clang 11.1.0 "clang version 11.1.0")
C++ linker for the host machine: clang++ ld64 609
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: /nix/store/jm59y1h4xx9gax9xjif8cy12jn5w04y6-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
qmake found: YES (/nix/store/svqp8cg96skk0cda5mzkkzmrji8v8dxj-qtbase-6.4.3-dev/bin/qmake) 6.4.3
Run-time dependency qt6 (modules: Core, Gui, OpenGL) found: YES 6.4.3 (qmake)
Run-time dependency luajit found: YES 2.1.0-beta3
Run-time dependency zlib found: YES 1.2.13
Run-time dependency gl found: YES
Detecting Qt6 tools
qmake found: YES (/nix/store/svqp8cg96skk0cda5mzkkzmrji8v8dxj-qtbase-6.4.3-dev/bin/qmake) 6.4.3
Run-time dependency qt6 (modules: Core) found: YES 6.4.3 (qmake)
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/bin/moc found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/libexec/moc found: NO
Program moc6 found: NO
Program moc-qt6 found: NO
Program moc found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/bin/uic found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/libexec/uic found: NO
Program uic6 found: NO
Program uic-qt6 found: NO
Program uic found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/bin/rcc found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/libexec/rcc found: NO
Program rcc6 found: NO
Program rcc-qt6 found: NO
Program rcc found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/bin/lrelease found: NO
Program /nix/store/v7b80icccsc9ng1ram1m1sa5xjj6pbpr-qtbase-6.4.3/libexec/lrelease found: NO
Program lrelease6 found: NO
Program lrelease-qt6 found: NO
Program lrelease found: YES 6.4.3 (/nix/store/kdr2hv5q666g2qngnxypvg6lv7ggfc9p-qttools-6.4.3-dev/bin/lrelease)
meson.build:23:0: ERROR: MOC sources specified and couldn't find uic-qt6, please check your qt6 installation
A full log can be found at /private/tmp/nix-build-path-of-building-2.27.2-unstable-2023-04-09.drv-0/source/build/meson-logs/meson-log.txt
```

Reply:
I guess. I don't think upstream cares about Darwin either.

Reply:
Upstream does seem to kinda care about Darwin, but I don't want to investigate this blindly, so marked broken for now.